.github/workflows/nightly-test-integrations-1.17.x.yml

```diff
@@ -102,7 +102,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        envoy-version: ["1.24.10", "1.25.9", "1.26.4", "1.27.0"]
+        envoy-version: ["1.25.11", "1.26.6", "1.27.2", "1.28.0"]
```

Comment:
I think for 1.17, we'll want to keep to the current minor range (we were originally hoping the timing would work to move to 1.28, but 1.17 GA is currently set to release w/ Envoy 1.27)

Reply:
OK, thanks for pointing that out. That's good news in a sense because it means that the 1.17.x PRs (#19274 and #19275) can stay at 1.27.x for now. I will wait for @david-yu to confirm before reverting.

Reply:
Np, and sorry for not being clearer when we chatted before. There's a bit of time-denormalization in this specific instance that made this change extra 🧠-melty for me.

Reply:
Yes based on our discussion earlier this week with engineering on the service mesh team, we agreed it is best to stay on 1.27.x for 1.17.x. Thank you.